services/spar/src/Spar/Scim/User.hs

```diff
@@ -495,20 +591,20 @@ updateValidScimUser tokinfo uid newValidScimUser =
 updateVsuUref ::
   TeamId ->
   UserId ->
-  ST.ValidExternalId ->
-  ST.ValidExternalId ->
+  AuthId ->
+  AuthId ->
   Spar ()
-updateVsuUref team uid old new = do
-  let geturef = ST.runValidExternalId Just (const Nothing)
+updateVsuUref _team uid old new = do
```

Comment:
```suggestion
updateVsuUref _team uid old new = do  -- TODO: remove `_team` from parameter list, it's unused.
```

Reply:
Why did you resolve this?  It was still in there, until now: 01f4473ad59b79994a1bfe1d0153b06d13cd21f1

Reply:
I'm sorry, I don't know. I resolve the suggestions after I create a local commit. <aybe I dropped the change during squashing/rebasing. I'll try a new a approach to prevent this from happening: Resolve only after looking at diff of the _pushed _ commits.

Reply:
i'm also fine with not resolving comments, and just merging/rebasing/squashing/whicheverwayresolving the PR eventually.  :)